enterprise/frontend/src/metabase-enterprise/whitelabel/index.js

```diff
@@ -77,6 +79,20 @@ if (hasPremiumFeature("whitelabel")) {
           options: getLoadingMessageOptions(),
           defaultValue: "doing-science",
         },
+        {
+          key: "help-link",
+          display_name: t`Help Link in the Settings menu`,
+          description: (
+            <p>
+              {t`The Settings menu includes a Help link that goes to `}
```

Comment:
This is a copy thing, but I don't think `Settings` should be capitalized. Can you ask design for confirmation?

Reply:
I asked Vitorio on figma